recipes/stone/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "stone" %}
+{% set version = "3.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dropbox/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 05a6c9fec0b923a5e54814f31365499d6e7b726aaddfe5b8ee2d1a4b6cb06d3a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - stone=stone.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.5
+  run:
+    - ply >=3.4
+    - python >=3.5
+    - six >=1.3.0
+    - __unix  # Currently blocked by https://github.com/dropbox/stone/issues/214
+
+test:
+  imports:
+    - stone
+    - stone.backends
+  commands:
+    - pip check
+    - stone --help
+    - pytest test
+  requires:
+    - pip
+    - pytest <5
+    - mock>=2.0.0,<5.0
+    - coverage==5.3
+  source_files:
+    - test
+
+about:
+  home: https://www.dropbox.com/developers
+  summary: Stone is an interface description language (IDL) for APIs.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  dev_url: https://www.dropbox.com/developers
+  doc_url: https://github.com/dropbox/stone/tree/main/docs
+
+extra:
+  recipe-maintainers:
+    - cjmartian
```

Comment:
Agreed in https://github.com/conda-forge/staged-recipes/pull/13972#issuecomment-780667351